### DISCUSSION TITLE
RSS feeds favicon

### DISCUSSION THREAD

**Discussion by caywin25** [FreshRSS/FreshRSS#6308] [Q&A] [answered]
Hello

What is freshrss based on to display rss feed favicons?
On some rss feeds I have the favicon and on others I don't (I only have the planet).
I find that having favicons on each feed makes it easier to navigate.

Thanks

**Comment by math-GH** [comment]
hi,
each feed should have its own favicon.

Could you please share feeds that does not display its favicon?

**Reply by caywin25** [reply]
Hi

For exemple : https://github.com/FreshRSS/FreshRSS/releases

**Reply by caywin25** [reply]
Normally there should be the github favicon I think

**Reply by math-GH** [reply]
ok, this feed should work for sure.

Do you have any entries in your browser console or any 4xx errors in the network tab there? (how to open the browser console in your browser: mouse right click somewhere on a website -> inspect -> see the tabs)

**Reply by caywin25** [reply]
I know I have upstream CSP rules on my proxy.
Maybe that's where the problem comes from?
But that would be surprising because some favicons are displayed good

**Reply by math-GH** [reply]
The browser console will tell you if there are reasons not to display images because of CSP

**Reply by caywin25** [reply]
The only message about CSP is : Content-Security-Policy: The page’s settings blocked the loading of a resource at data: (“default-src”).

**Reply by caywin25** [reply]
It's really strange I just put another github rss feed and I have the favicon

**Comment by Alkarex** [comment]
Try to force refresh the image

**Reply by caywin25** [reply]
Hi
Where can I find this option please ?
Thx

**Comment by caywin25** [comment, marked as answer]
Ok the problem seems resolved, I just did a clean installation of my instance.
It is possible that the problem comes from: #6288

Everything works perfectly.
Thanks :)
